### PR TITLE
fix(dashboard): Handle scaling the devices issues widget

### PR DESCRIPTION
**Pull request by oleorhagen**
This simply adds a few CSS entries to handle scaling both the to small and to
the big.

Ticket: None
Changelog: None

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: src/js/components/dashboard/widgets/baseWidget.js [hunk 1]
```diff
@@ -4,9 +4,11 @@ export const BaseWidget = ({ className = '', header, innerRef, main, onClick })
   <div className={`widget ${className}`} onClick={onClick} ref={ref => (innerRef ? (innerRef.current = ref) : null)}>
     {!!header && <div className="flexbox widgetHeader">{header}</div>}
     {React.isValidElement(main) ? (
-      <div className="widgetMainContent">{main}</div>
+      <div style={{ justifyContent: 'space-evenly' }} className="widgetMainContent">
+        {main}
+      </div>
     ) : (
-      <div className="flexbox column widgetMainContent">
+      <div style={{ maxHeight: '3em', maxWidth: '3em' }} className="flexbox column widgetMainContent">
```
Comment (mzedel) [inline src/js/components/dashboard/widgets/baseWidget.js:11]: https://github.com/mendersoftware/gui/blob/master/src/js/components/dashboard/widgets/actionabledevices.js#L47-L48 <- might be the proper place to set this

File: src/js/components/dashboard/widgets/baseWidget.js [hunk 2]
```diff
@@ -4,9 +4,11 @@ export const BaseWidget = ({ className = '', header, innerRef, main, onClick })
   <div className={`widget ${className}`} onClick={onClick} ref={ref => (innerRef ? (innerRef.current = ref) : null)}>
     {!!header && <div className="flexbox widgetHeader">{header}</div>}
     {React.isValidElement(main) ? (
-      <div className="widgetMainContent">{main}</div>
+      <div style={{ justifyContent: 'space-evenly' }} className="widgetMainContent">
```
Comment (mzedel) [inline src/js/components/dashboard/widgets/baseWidget.js:7]: https://github.com/mendersoftware/gui/blob/master/src/js/components/dashboard/widgets/actionabledevices.js#L85 <- might be the proper place to set this

**Review by mzedel — CHANGES_REQUESTED** [review]

File: src/js/components/dashboard/widgets/baseWidget.js (hunk 2 above)
Comment (oleorhagen) [reply src/js/components/dashboard/widgets/baseWidget.js:7]: Ohh, I see it now, thx :)

**Review by mzedel — CHANGES_REQUESTED** [review]
to make the unit tests pass you'll have to update the related snapshots... in order to do so you can either run `npm run test -- --watchAll -u` or `npm run test -- --watch <someFileToBeUpdatedHere> -u`
words of warning:
- make sure to run the tests using node v14 (there is some more work needed to ensure all of them work with later node versions 🙈)
- if going with the `watchAll` route you might want to have something to read ready, as this can take a bit of your CPU cycles 😄

**Review by mzedel — APPROVED** [review]

**Pull request merged**